azkaban-web-server/src/main/resources/azkaban/webapp/servlet/velocity/svgflowincludes.vm

```diff
@@ -31,9 +31,9 @@
 
 <script type="text/javascript" src="${context}/js/azkaban/util/flow-loader.js?v=1620232349"></script>
 <script type="text/javascript" src="${context}/js/azkaban/util/job-list-common.js?v=1570835891"></script>
-<script type="text/javascript" src="${context}/js/azkaban/view/job-list.js?v=1570835891"></script>
-<script type="text/javascript" src="${context}/js/azkaban/model/svg-graph.js?v=1620232349"></script>
+<script type="text/javascript" src="${context}/js/azkaban/view/job-list.js?v=1623085074"></script>
```

Comment:
We're doing this for the dependencies altered below, but is this the only file we need to do this in, and do we not need to do something similar for svg-graph?

Reply:
Sorry, I missed the third change.

Reply:
No prob.